config/locales/en.json

```diff
@@ -504,11 +504,18 @@
 	"dashboard.overall-health": "Patient's Overall Health",
 	"dashboard.expedited-processing": "Expedited processing for terminal and grave conditions",
 	"conditions.incomplete": "Not Completed",
+	"invite.expires": "This code will expire on %s",
+	"invite.code_loss_instruction": "If you lose your code, or it expires, fill out this form again to generate another code",
```

Comment:
Missing a full stop/period on the end of the sentence.

Reply:
done